lib/tasks/heroku.rake

```diff
@@ -0,0 +1,59 @@
+namespace :heroku do
+  desc 'Runs once on the first PR deploy to Heroku, used in app.json'
+  task :post_deploy do
+    require 'excon'
+
+    sso_url = ENV.fetch('MOJSSO_URL')
+
+    new_app_name = ENV.fetch('HEROKU_APP_NAME')
+
+    service_uri = "https://#{new_app_name}.herokuapp.com"
+
+    post_data = {
+      'app_name' => 'Prison Visits Booking',
```

Comment:
We can, I assumed Staging and Production would have the same app name for PVB. How come they are different?